### PR TITLE
Run Cypress tests on Keycloak theme

### PR DESCRIPTION
**Pull request by jonkoops**
## Motivation
Allows us to run the tests against the final product instead of our own dev server.

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -1,53 +1,50 @@
...
+        run: docker build --tag keycloak .
+
+      - name: Start Keycloak server
+        run: |
+          docker run --detach --publish 8080:8080 --env KEYCLOAK_USER=admin --env KEYCLOAK_PASSWORD=admin --name keycloak keycloak
+          sleep 10
+
+      - name: Configure Keycloak server
+        run: |
+          docker exec keycloak /opt/jboss/keycloak/bin/kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin --password admin
```
Comment (edewit) [inline .github/workflows/cypress.yml:23]: this is not needed as you start with `KEYCLOAK_USER`

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -1,53 +1,50 @@
...
-        path: assets/screenshots
\ No newline at end of file
+      - uses: actions/checkout@v2
+
+      - name: Build Keycloak server
+        run: docker build --tag keycloak .
+
+      - name: Start Keycloak server
+        run: |
+          docker run --detach --publish 8080:8080 --env KEYCLOAK_USER=admin --env KEYCLOAK_PASSWORD=admin --name keycloak keycloak
```
Comment (edewit) [inline .github/workflows/cypress.yml:18]: don't we need to add the flag? `-Dprofile.feature.newadmin=enabled`

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (jonkoops) [reply .github/workflows/cypress.yml]: Good catch, I've removed this.

File: .github/workflows/cypress.yml (hunk 2 above)
Comment (jonkoops) [reply .github/workflows/cypress.yml:18]: Yeah, although I am not sure how.

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (jonkoops) [reply .github/workflows/cypress.yml:23]: Hmm, this causes an issue where it cannot reach the API. I'll leave it for now.

File: .github/workflows/cypress.yml (hunk 2 above)
Comment (jonkoops) [reply .github/workflows/cypress.yml:18]: Added this as an argument.

**Review by edewit — APPROVED** [review]

**Pull request merged**